### PR TITLE
Add more keys to briefs data that gets exported to csv

### PR DESCRIPTION
**Pull request by CrystalPea**
Trello card: https://trello.com/c/sdwj5wKt/275-add-more-details-to-automatic-data-export-script

### CODE REVIEW

**Comment by benvand** [comment]
LGTM
This method https://github.com/alphagov/digitalmarketplace-scripts/blob/master/dmscripts/models/modeltrawler.py#L31-L31
makes sure that if the key doesn't exist then `''` is returned instead of the value.

Running this:
```
from dmapiclient import DataAPIClient
client = DataAPIClient('http://localhost:5000', 'myToken')
briefs = client.find_briefs()['briefs']
counter = dict(zip(['frameworkSlug', 'budgetRange', 'startDate', 'contractLength'], [0 for _ in range(4)]))
for brief in briefs:                                                                                       
     for v in ['frameworkSlug', 'budgetRange', 'startDate', 'contractLength']:
         try:
             print brief[
             counter[v] += 1
         except:
             pass
print counter

Out: 
{'budgetRange': 55,
 'contractLength': 85,
 'frameworkSlug': 100,
 'startDate': 95}
```
Shows that the fields are legit.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 *.pyc
 __pycache__
 output/
+venv
+venv3
```
Comment (idavidmcdonald) [inline .gitignore]: Either way works but same outcome could be achieved by `venv*`

**Comment by idavidmcdonald** [comment]
👍 

I left a optional comment about venv things but apart from that it looks good.

**Comment by idavidmcdonald** [comment]
Ah, sorry misunderstanding.
If you have 
`venv*` in your .gitignore then you don't also need to have `venv`.
If you pop a quick pull request in to fix that and then I'll thumbs it.

You might also be interested in reading a bit more about clever ways of gitignoring stuff - http://365git.tumblr.com/post/519016351/three-ways-of-excluding-files

**Pull request merged**